apps/condo/domains/ticket/schema/TicketFile.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Generated by `createschema ticket.TicketFile 'organization:Text;file?:File;ticket?:Relationship:Ticket:SET_NULL;'`
+ */
+
+const { Relationship, File } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/TicketFile')
+const { ORGANIZATION_OWNED_FIELD } = require('../../../schema/_common')
+
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const Adapter = new FileAdapter('ticket')
+
+// zuch(todo): find a way to upload images in jest tests
+const TicketFile = new GQLListSchema('TicketFile', {
+    schemaDoc: 'File attached to the ticket',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+        organization: ORGANIZATION_OWNED_FIELD,
+        file: {
+            schemaDoc: 'File object with meta information and publicUrl',
+            type: File,
+            adapter: Adapter,
+            isRequired: false,
```

Comment:
Why it’s not required ?

Reply:
Because I can't make jest tests to test ticket file main functionality without switching require off.  I have written a to-do record about this fact. If you have any idea how to emulate browser forms behavior from jest tests or how to mock file field - any suggestions are welcome.

Reply:
File field is a json data. Can you use a fake json data?